lib/osf-components/addon/components/node-metadata-form/template.hbs

```diff
@@ -255,17 +256,90 @@
-                        {{#each @manager.institutions as |institution|}}
-                            <dd>{{institution.name}}</dd>
-                        {{/each}}
-                    </dl>
-                </div>
-            {{/if}}
+            <div data-test-target-institutions>
+                {{#if @manager.isEditingInstitutions}}
+                    <div local-class='metadata-header'>
+                        {{t 'osf-components.node-metadata-form.institutions'}}
+                    </div>
+                    <InstitutionSelectList @manager={{@manager}} />
+                    <Button
+                        aria-label={{t 'general.cancel'}}
+                        data-analytics-name='Cancel editing node institutions'
+                        data-test-cancel-editing-node-institutions-button
+                        disabled={{@manager.isSaving}}
+                        {{on 'click' @manager.cancelInstitutions}}
+                        @type='secondary'
+                    >
+                        {{t 'general.cancel'}}
+                    </Button>
+                    <Button
+                        aria-label={{t 'general.save'}}
+                        data-analytics-name='Save node institutions'
+                        data-test-save-node-institutions-button
+                        disabled={{@manager.isSaving}}
+                        {{on 'click' @manager.saveInstitutions}}
+                        @type='primary'
+                    >
+                        {{t 'general.save'}}
+                    </Button>
+                {{else}}
+                    <div local-class='metadata-header'>
+                        {{t 'osf-components.node-metadata-form.institutions'}}
+                        {{#if (and @manager.userCanEdit (not @manager.inEditMode))}}
+                            <Button
+                                data-test-edit-institutions-button
+                                data-analytics-name='Edit Institutions'
+                                aria-label={{t 'osf-components.node-metadata-form.edit-button'}}
+                                {{on 'click' @manager.editInstitutions}}
+                                @layout='fake-link'
+                            >
+                                <FaIcon @icon='pencil-alt' />
+                            </Button>
+                        {{/if}} 
+                    </div>
+                    <InstitutionsList @manager={{@manager}} />
+                {{/if}}
+            </div>
         {{/unless}}
+        <dl>
+            <dt>
+                <span>
+                    {{t 'osf-components.node-metadata-form.dates'}}
+                </span>
+            </dt>
+            <dd data-test-creation-date>
+                {{t 'osf-components.node-metadata-form.creation-date'}}
+                {{moment-format @manager.node.dateCreated 'MMMM D, YYYY'}}
+            </dd>
```

Comment:
The same suggestion as I had for the guid-file template's date stuff applies here. `<dt>` seems more appropriate as a heading, `<dd>` seems more appropriate being split into 
```
<dt>{{t 'osf-components.node-metadata-form.creation-date'}}</dt>
<dd>{{moment-format @manager.node.dateCreated 'MMMM D, YYYY'}}</dd>
```